improver/developer_tools/run_metadata_interpreter.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
This script doesn't run out-of-the-box the way other IMPROVER CLIs do. The reason is that it is not linked up with the etc/site-init file, so doesn't know anything about importing the scitools stack, and has no interpreter defined at the start of the script, so shell doesn't know what the intention is. 
We discussed moving this, or perhaps calling this from, a script in bin where it can use some of the centrally-supported IMPROVER architecture. I think this would be highly desirable.

Reply:
I agree.  Will come back to this aspect, or move to a follow-on ticket if necessary.

Reply:
Removed script and defined follow-on work to achieve the desired behaviour.